app/views/claims/support/claims/_filter.html.erb

```diff
@@ -0,0 +1,153 @@
+<div class="app-filter-layout">
+  <div class="app-filter-layout__filter">
+    <div class="app-filter__header">
+      <div class="app-filter__header-title">
+        <h2 class="govuk-heading-m"><%= t("filter") %></h2>
+      </div>
+      <div class="app-filter__header-action"></div>
+    </div>
+    <div class="app-filter-layout__content">
+      <% if @filter_form.filters_selected? %>
+        <div class="app-filter-layout__selected">
+          <div class="app-filter__selected-heading">
+            <div class="app-filter__heading-title">
+              <h2 class="govuk-heading-m govuk-!-margin-bottom-1"><%= t("selected_filters") %></h2>
+              <p class="govuk-body">
+                <%= govuk_link_to(
+                  t("clear_filters"),
+                  @filter_form.clear_filters_path,
+                  no_visited_state: true,
+                ) %>
+              </p>
+            </div>
+          </div>
+
+          <% if @filter_form.school_ids.present? %>
+            <h3 class="govuk-heading-s govuk-!-margin-bottom-0"><%= t("claims.support.claims.index.school") %></h3>
+            <ul class="app-filter-tags">
+              <% @filter_form.school_ids.each do |school_id| %>
+                <li>
+                 <%= govuk_link_to(
+                   Claims::School.find_by_id(school_id)&.name,
```

Comment:
This too. Let's avoid N+1 queries.

Reply:
We should be using `filter_form.schools` here, right?

Reply:
Stashed the wrong changes